src/Components/Footer.js

```diff
@@ -102,6 +119,9 @@ function Footer() {
         </FirstColumn>
         <SecondColumn>
           <Navigation>
+            {/* For things like this, it can be nice to make an array of objects representing
+            each link and then map over it instead of listing them all out, just to save space.
+            Same thing for the socials and contacts below */}
```

Comment:
@jahorwitz can I keep an array of objects in this file or should I move it to utils/data.js where the rest of the data is?